### ISSUE TITLE
Replace (bad) preview policy about Storage account public accessibility

### ISSUE THREAD

**Issue by TriP00r** [Azure/Bio-Compliancy#16]
The policy 4fa4b6c0-31ca-4c0d-b10d-24b96f62a751 ([Preview]: Storage account public access should be disallowed) should be replaced with f4ac74bb-59d1-42ee-a7fb-e9b9f525fb03 (Storage account public access should be disallowed (block anonymous blob access)).
The former is incorrecly written so it reports when public access is denied, instead of when it is allowed.

**Comment by PieterbasNagengast** [comment]
Hi @TriP00r
Thanks for creating this issue. Not sure if I understand you correctly, could you please be more specific?

**Comment by TriP00r** [comment]
Hi @PieterbasNagengast ,

Thanks for your reply.
I found the solution and was confused by the policy definition where it states:
{
            "not": {
              "field": "Microsoft.Storage/storageAccounts/allowBlobPublicAccess",
              "equals": "false"
            }
          }

So in my head allowBlobPublicAccess should not be equal to false. So that means true. Incidentally the result was the same as the setting Network > Public Access > Disabled for all our resources, and the one where it deviated this was enabled. I took the wrong conclusion. Therefor I suggested another policy.
I fixed it in our environment and understand the policy better, this issue can be closed.

Thanks for your time!
